src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjectById.cs

```diff
@@ -36,11 +36,9 @@ public GetProjectById(IProjectsService projectsService)
             }
             catch (ProjectsException e)
             {
-                return new ContentResult
-                {
-                    StatusCode = e.StatusCode,
-                    Content = e.ProjectsErrorMessage,
-                };
+                var error = new ObjectResult(e.ErrorMessage);
```

Comment:
Can this ObjectResult be created inside ProjectsException? that way we only need to return a property instead of doing this logic for every exception in the code

Reply:
No it cannot, I need a using and is for azure function only

Reply:
DONE, Thanks Raul!